packages/ssz/src/backings/tree/list.ts

```diff
@@ -0,0 +1,197 @@
+import {Node, Tree, zeroNode, BranchNode} from "@chainsafe/persistent-merkle-tree";
+
+import {List} from "../../interface";
+import {number32Type, BasicListType, CompositeListType} from "../../types";
+import {BasicArrayTreeHandler, CompositeArrayTreeHandler} from "./array";
+
+export class BasicListTreeHandler<T extends List<unknown>> extends BasicArrayTreeHandler<T> {
+  _type: BasicListType<T>;
+  _defaultNode: Node;
+  constructor(type: BasicListType<T>) {
+    super();
+    this._type = type;
+  }
+  defaultNode(): Node {
+    if (!this._defaultNode) {
+      this._defaultNode = new BranchNode(zeroNode(super.depth()), zeroNode(0));
+    }
+    return this._defaultNode;
+  }
+  defaultBacking(): Tree {
+    return new Tree(this.defaultNode());
+  }
+  getLength(target: Tree): number {
+    return number32Type.fromBytes(target.getRoot(BigInt(3)), 0);
+  }
+  setLength(target: Tree, length: number): void {
+    const chunk = new Uint8Array(32);
+    number32Type.toBytes(length, chunk, 0);
+    target.setRoot(BigInt(3), chunk);
+  }
+  fromBytes(data: Uint8Array, start: number, end: number): Tree {
+    const length = (end - start) / this._type.elementType.size();
```

Comment:
You may want to handle the case where `end - start` is not a multiple of element size.